### PR TITLE
SEID2 Support

### PR DESCRIPTION
**Pull request by runeflobakk**
Enables the client to accept Posten signering API to use the new SEID2 standard for its enterprise certificate.

### CODE REVIEW

File: src/main/java/no/digipost/signature/client/security/CertificateChainValidation.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import javax.net.ssl.SSLContext;
+
+import java.security.cert.X509Certificate;
+
+public interface CertificateChainValidation {
+
+    enum Result {
+        /**
+         * Indicates that the certificate chain is trusted by this particular
+         * validation, but is subject to furher validation by the {@link SSLContext}'s
```
Comment (simenstoa) [inline src/main/java/no/digipost/signature/client/security/CertificateChainValidation.java:12]:
```suggestion
         * validation, but is subject to further validation by the {@link SSLContext}'s
```

File: src/test/java/no/digipost/signature/client/security/OrganizationNumberValidationTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import static no.digipost.signature.client.security.CertificateChainValidation.Result.TRUSTED;
+import static no.digipost.signature.client.security.CertificateChainValidation.Result.UNTRUSTED;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static uk.co.probablyfine.matchers.Java8Matchers.where;
+
+public class OrganizationNumberValidationTest {
+
+    @Test
+    void trusts_Posten_Norge_SEID1_enterprise_certificate() {
```
Comment (simenstoa) [inline src/test/java/no/digipost/signature/client/security/OrganizationNumberValidationTest.java:17]: Skulle vi hatt en tilsvarende test for SEID2?

**Review by simenstoa — APPROVED** [review]
💯

File: src/test/java/no/digipost/signature/client/security/OrganizationNumberValidationTest.java (hunk 2 above)
Comment (runeflobakk) [reply src/test/java/no/digipost/signature/client/security/OrganizationNumberValidationTest.java]: Jepp, la det til her 8a89ac4

**Pull request merged**